### ISSUE TITLE
Does it only support serial/rs232 connections?

### ISSUE THREAD

**Issue by justingoldberg** [ekovac/x86term#3]
I know that it's a dumb question, but does it support anything beyond serial/rs232 such as ethernet connections?

Will it run on an intel 486?

**Comment by ekovac** [comment]
Nope, it doesn't support anything other than serial/rs232. I have some thoughts about revisiting this project at some point but I don't think ethernet would be in-scope for this. Doing so would require implementing a network card driver (which one?), an IP/TCP stack, a DHCP client, etc.) Unless you're referring to some more esoteric use of ethernet?

Would maybe be cool to support enough functionality for dialing into a terminal server via a serial modem though, that'd require a lot less device-specific work.

I believe it *should* run on an Intel 486 PC, or even a 386, but I've not tested it. Ultimately any PC-compatible thing should work.
